src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -27,12 +27,17 @@ using namespace dnnl::impl::cpu::x64;
 using namespace Xbyak;
 
 template <typename src_t, typename dst_t>
-void convert_vec(jit_generator & gen,
+void convert_vec_avx2(jit_generator & gen,
+                 const RegExp & src,
+                 const RegExp & dst);
+
+template <typename src_t, typename dst_t>
+void convert_vec_avx512(jit_generator & gen,
```

Comment:
Probably we can use ISA as the template parameter here. Refer to https://github.com/ceciliapeng2011/openvino/blob/cecilia/kvcache_fp16/src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp